src/makeCalculator.js

```diff
@@ -4,7 +4,31 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  return {
+    add: (num) => (result += num),
+
+    subtract: (num) => (result -= num),
+
+    multiply: (num) => (result *= num),
+
+    divide: (num) => (result /= num),
+
+    operate: function (callback, num) {
```

Comment:
According to the checklist, when creating a method in an object, you should use the shortcut syntax instead of the 'function' keyword. You can define the 'operate' method as 'operate(callback, num) { ... }'.